contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -1005,6 +1005,47 @@ fooRef:
 
 Although not always necessary to help a controller identify a resource type, “group” is included to avoid ambiguity when the resource exists in multiple groups. It also provides clarity to end users and enables copy-pasting of a reference without the referenced type changing due to a different controller handling the reference.
 
+##### Kind vs. Resource
```

Comment:
this can be a follow-up PR (and you don't have to be the one to do it either), but this could use a sentence addressing why basically every core Kubernetes object reference uses kind. 

I think we just need to say it's a legacy behavior, and in the future we'll look at using resource-style instead. But I might be making assumptions here, and we probably want @deads2k or @lavalamp  to weigh in on that.

Reply:
I agree that would be helpful. I don't think I have the historical context to add something here, but would appreciate any follow ups on this front.

Reply:
I think this could be accomplished by adding to line 1027 "(this is the case for built-in references which use Kind)".

Reply:
Updated, thanks!